src/__swaps__/screens/Swap/hooks/useAssetsToSell.ts

```diff
@@ -1,58 +0,0 @@
-import { useMemo } from 'react';
-
-import {
-  selectUserAssetsList,
-  selectUserAssetsListByChainId,
-  selectorFilterByUserChains,
-} from '@/__swaps__/screens/Swap/resources/_selectors/assets';
-import { useUserAssets } from '@/__swaps__/screens/Swap/resources/assets';
-import { ParsedAssetsDictByChain, ParsedSearchAsset, UserAssetFilter } from '@/__swaps__/types/assets';
-import { useAccountSettings, useDebounce } from '@/hooks';
-import { useUserAssetsStore } from '@/state/assets/userAssets';
-
-const sortBy = (by: UserAssetFilter) => {
-  switch (by) {
-    case 'all':
-      return selectUserAssetsList;
-    default:
-      return (data: ParsedAssetsDictByChain) => selectUserAssetsListByChainId(data, by);
-  }
-};
-
-export const useAssetsToSell = () => {
```

Comment:
deleting since this wasn't being used anywhere

Reply:
fwiw this is removed in swaps v1 cleanup PR